src/main/kotlin/com/github/pool_party/pull_party_bot/commands/messages/DeleteMessages.kt

```diff
@@ -4,17 +4,33 @@ val ON_CLEAR_SUCCESS =
     """
     Life is a party. Chat is dead then. 😭
 
-    All parties are over. Hope you did it only to rock even bigger ones
-    Let's rest just a little and I'll be waiting for your new entries
+    All parties are over. Hoping you did it only to rock even bigger ones!
+    Have a break and I'll be waiting for your new entries
     Use /create command to start all over again
 
     Type `/help create` for more information
     """.trimIndent()
 
-fun onPartyDeleteSuccess(partyName: String) = """Party $partyName is just a history now 👍"""
+fun onAliasDeleteSuccess(partyName: String) = """Alias $partyName is no longer exists 👍"""
+
+fun onPartyDeleteSuccess(partyName: String) = """Party $partyName and its aliases are deleted 👍"""
+
+fun onPartyDeleteSuggest(partyList: List<String>): String {
+    val message = """Perhaps you want to delete all aliases of"""
+
+    val format =
+        when (partyList.size) {
+            0 -> """"""// TODO check if case is possible.
```

Comment:
huh?

Reply:
I were not sure, can be removed if there is a contract of nonempty list

Reply:
Well, there is a contract that if the last alias is deleted, then the party itself is deleted, as well. So there cannot be an empty party list, afaik 🙃